framework/OpenMod.Core/Commands/CommandExecutor.cs

```diff
@@ -63,15 +63,15 @@ public async Task<ICommandContext> ExecuteAsync(ICommandActor actor, string[] ar
                 return commandExecutingEvent.CommandContext;
             }
 
-            logger.LogInformation($"Actor {actor.Type}/{actor.DisplayName} ({actor.Id}) has executed command \"{string.Join(" ", args)}\".");
-
             try
             {
                 if (commandContext.Exception != null)
                 {
                     throw commandContext.Exception;
                 }
 
+                //The console entry was already logged, this fixes the wrong message when the command is not found
+                logger.LogInformation($"Actor {actor.Type}/{actor.DisplayName} ({actor.Id}) has executed command \"{string.Join(" ", args)}\".");
```

Comment:
This was done on purpose, must be logged *before* exception. Otherwise commands causing CommandWrongUsageException and other exceptions will not log. It should also log commands which were not found,